src/hooks/useCookieConsent.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react'
+
+export const useCookieConsent = () => {
```

Comment:
Previous behaviour was to only add cookie consent on prod builds - can we update this to be the same behaviour? ie so it doesn't run by default in dev envs.